pkg/tools/dnsutils/dnsconfigs/handler.go

```diff
@@ -44,7 +44,7 @@ func (h *dnsConfigsHandler) ServeDNS(ctx context.Context, rp dns.ResponseWriter,
 		for _, conf := range value {
 			ips := make([]url.URL, len(conf.DnsServerIps))
 			for i, ip := range conf.DnsServerIps {
-				ips[i] = url.URL{Scheme: "tcp", Host: ip}
+				ips[i] = url.URL{Scheme: "udp", Host: ip}
```

Comment:
I think we should support both schemes, because we don't know the right scheme

ips := make([]url.URL,0, len(conf.DnsServerIps) * 2)
for i, ip := range conf.DnsServerIps {
				ips = append(ips, &url.URL{Scheme: "udp", Host: ip}
				ips = append(ips, &url.URL{Scheme: "tcp", Host: ip}
}

Reply:
Added